### PR TITLE
Config for Optimism Sepolia

### PR DESCRIPTION
**Pull request by kovalgek**
Replace Goerli testnet configs and scripts by Sepolia

### CODE REVIEW

File: .env.wsteth.opt_sepolia [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+L2_WITHDRAWALS_ENABLERS=["0xf695357C66bA514150Da95b189acb37b46DDe602"]
+L2_WITHDRAWALS_DISABLERS="["0xf695357C66bA514150Da95b189acb37b46DDe602", "0xa5F1d7D49F581136Cf6e58B32cBE9a2039C48bA1"]"
+
+# ############################
+# Integration & E2E Testing
+# ############################
+
+TESTING_OPT_NETWORK=sepolia
+TESTING_OPT_L1_TOKEN=0xB82381A3fBD3FaFA77B3a7bE693342618240067b
+TESTING_OPT_L2_TOKEN=
```
Comment (Psirex) [inline .env.wsteth.opt_sepolia:56]: I would also put the addresses of the deployed contracts here

File: hardhat.config.ts [hunk 2]
```diff
@@ -91,12 +91,31 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: {
       mainnet: env.string("ETHERSCAN_API_KEY_ETH", ""),
-      goerli: env.string("ETHERSCAN_API_KEY_ETH", ""),
-      arbitrumGoerli: env.string("ETHERSCAN_API_KEY_ARB", ""),
+      sepolia: env.string("ETHERSCAN_API_KEY_ETH", ""),
+      // arbitrumSepolia: env.string("ETHERSCAN_API_KEY_ARB", ""),
```
Comment (Psirex) [inline hardhat.config.ts:95]: Maybe remove this line, if it is not used? Or worth considering adding the comment, on why this line is kept

File: hardhat.config.ts [hunk 3]
```diff
@@ -91,12 +91,31 @@ const config: HardhatUserConfig = {
...
       arbitrumOne: env.string("ETHERSCAN_API_KEY_ARB", ""),
       optimisticEthereum: env.string("ETHERSCAN_API_KEY_OPT", ""),
-      optimisticGoerli: env.string("ETHERSCAN_API_KEY_OPT", ""),
+      "opt_sepolia": env.string("ETHERSCAN_API_KEY_OPT", ""),
```
Comment (Psirex) [inline hardhat.config.ts:98]: The repo uses "prettier" for code formatting, but currently, it's not configured to work with pre-commit hook (for example, https://github.com/lint-staged/lint-staged?tab=readme-ov-file). Consider adding pre-commit linting for the files (at least typescript files) and enabling auto-formatting in the IDE.

File: test/optimism/bridging.e2e.test.ts [hunk 4]
```diff
@@ -109,6 +110,9 @@ scenario("Optimism :: Bridging via deposit/withdraw E2E test", ctxFactory)
   })
 
   .step("Finalizing L2 -> L1 message", async (ctx) => {
+    const safetyDelay = 1000;
```
Comment (Psirex) [inline test/optimism/bridging.e2e.test.ts:113]: Unused variable

**Review by Psirex — APPROVED** [review]
LGTM 👍🏻 Left only some minor comments

**Review by arwer13 — APPROVED** [review]
LGTM, except for the @Psirex comments

**Pull request merged**